### PR TITLE
English Check

### PR DESCRIPTION
**Pull request by EliChin**
Checked for some inaccuracies regarding English.

Signed-off-by: Elizaveta Chichindaeva <elizaveta@nspcc.ru>

### CODE REVIEW

File: cmd/neofs-adm/README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@
 Admin tool provides an easier way to deploy and maintain private installation
 of NeoFS. Private installation provides a set of N3 consensus nodes, NeoFS 
 Alphabet, and Storage nodes. Admin tool generates consensus keys, initializes 
-side chain, and provides functions to update the network and register new
+the sidechain, and provides functions to update the network and register new
```
Comment (carpawell) [inline cmd/neofs-adm/README.md:8]: could you, please, provide the rule describing the difference b/w "side chain", "the sidechain" and "side chain"? see all in that PR

Comment (EliChin) [reply cmd/neofs-adm/README.md:8]: I've discussed it with @realloc and we've decided to write `sidechain`, `mainchain` etc in one word when it is one entity (well, in our context everywhere). Definite article is used with an entity that is the only one in a certain context, so I'd recommend to use `the` with `sidechain`. Unless `sidechain` is an attribute to some noun, eg. `sidechain contract` -- here we use article as needed for the noun (a ... contract, the .. contract, ... contracts)

File: CHANGELOG.md [hunk 2]
```diff
@@ -339,7 +339,7 @@ NeoFS API v2.10 support
 - Detailed comments in neofs-node config example (#858)
 - Size suffixes support in neofs-node config (#857)
 - Docs for neofs-adm (#906)
-- Side chain block size duration and global NeoFS configuration in
+- side chain block size duration and global NeoFS configuration in
```
[the remark this answers is not in the document]
Comment (EliChin) [reply CHANGELOG.md:342]: It is my typo here. 
Now, I'm not sure what to do with CHANGELOG.md files. I'll check this file later.

File: cmd/neofs-adm/docs/subnetwork-creation.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # NeoFS subnetwork creation
 
 This is a short guide on how to create NeoFS subnetworks. This guide 
-considers that side chain and inner ring (alphabet nodes) have already 
-been deployed and side chain contains deployed `subnet` contract.
+considers that the sidechain and the inner ring (alphabet nodes) have been already
```
Comment (fyrchik) [inline cmd/neofs-adm/docs/subnetwork-creation.md:4]: Why, though? I though `have already ...` is more common. Though I have nor arguments except for this random link https://textranch.com/176789/have-been-already/or/have-already-been/

File: cmd/neofs-adm/docs/subnetwork-usage.md [hunk 4]
```diff
@@ -81,17 +81,17 @@ $ neofs-adm morph subnet client add \
 Add client request sent successfully.
 ```
 
-**NOTE:** owner of the subnetwork is also allowed to add clients. This is
+**NOTE:** the owner of the subnetwork is also allowed to add clients. This is
 the only one command that accepts `ownerID`, not the public key.
-Administrator can manage only his group (a group where that administrator
-has been added by subnet owner).
+Administrator can manage only its group (a group where that administrator
```
Comment (fyrchik) [inline cmd/neofs-adm/docs/subnetwork-usage.md:86]: `their`?

File: cmd/neofs-node/config/contracts/config.go [hunk 5]
```diff
@@ -11,47 +11,47 @@ const (
...
-// Returns zero filled script hash if value is not set.
-// Throws panic if value is not a 20-byte LE hex-encoded string.
+// Returns zero filled script hash if the value is not set.
+// Throws panic if the value is not a 20-byte LE hex-encoded string.
 func Balance(c *config.Config) util.Uint160 {
 	return contractAddress(c, "balance")
 }
 
-// Container returns value of "container" config parameter
+// Container returnsthe  value of "container" config parameter
```
Comment (fyrchik) [inline cmd/neofs-node/config/contracts/config.go:32]: Missing space here and a couple of lines below.

File: pkg/innerring/innerring.go [hunk 6]
```diff
@@ -198,7 +198,7 @@ func (s *Server) Start(ctx context.Context, intError chan<- error) (err error) {
 	prm := governance.VoteValidatorPrm{}
 	prm.Validators = s.predefinedValidators
 
-	// vote for sidechain validator if it is prepared in config
+	// vote for side chain validator if it is prepared in config
```
Comment (fyrchik) [inline pkg/innerring/innerring.go:201]: It is without space every where, so why a space here?

File: cmd/neofs-adm/docs/subnetwork-usage.md (hunk 4 above)
Comment (EliChin) [reply cmd/neofs-adm/docs/subnetwork-usage.md:86]: Personally, i would use `their`, but it's not quite common in our repos. But, if you supports this option, I would change it :)

File: cmd/neofs-adm/docs/subnetwork-creation.md (hunk 3 above)
Comment (EliChin) [reply cmd/neofs-adm/docs/subnetwork-creation.md:4]: For some reason, I've always tended to use it like this. Now, I've explored the issue and probably `have already been...` is better

File: pkg/core/container/fmt.go [hunk 7]
```diff
@@ -18,7 +18,7 @@ var (
 // CheckFormat conducts an initial check of the v2 container data.
 //
 // It is expected that if a container fails this test,
-// it will not be inner-ring approved.
+// it will not be approved by the inner-ring.
```
Comment (anatoly-bogatyrev) [inline pkg/core/container/fmt.go:21]: Could you please change inner-ring to `inner ring` here? Thanks.

**Review by anatoly-bogatyrev — APPROVED** [review]

File: pkg/core/container/fmt.go (hunk 7 above)
Comment (EliChin) [reply pkg/core/container/fmt.go]: Done

File: README.md [hunk 8]
```diff
@@ -14,13 +14,13 @@
...
 and distributing user's data. Any Neo user may participate in the network and
-get paid for providing storage resources to other users or store his data in
+get paid for providing storage resources to other users or store their data in
 NeoFS and pay a competitive price for it.
 
 Users can reliably store object data in the NeoFS network and have a transparent
 data placement process due to a decentralized architecture and flexible storage
 policies. Each node is responsible for executing the storage policies that the
-users select for geographical location, reliability level, number of nodes, type
+users selects for geographical location, reliability level, number of nodes, type
```
Comment (carpawell) [inline README.md:23]: why with "s"?

File: cmd/neofs-adm/internal/modules/morph/internal/types.go [hunk 9]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // StringifySubnetClientGroupID returns string representation of SubnetClientGroupID using MarshalText.
-// Returns string with message on error.
+// It returns a string with a message on error.
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/morph/internal/types.go:11]: is it really necessary? i mean i know that it is correct but we have that pattern almost everywhere just to keep decs shorter i guess
will be happy to hear some recommendations about func docs (it always should contain repetitions like "It returns... It panics... It does... It does not..." then)

File: cmd/neofs-node/config/engine/config.go [hunk 10]
```diff
@@ -47,9 +47,9 @@ func IterateShards(c *config.Config, required bool, f func(*shardconfig.Config))
 	}
 }
 
-// ShardPoolSize returns value of "shard_pool_size" config parameter from "storage" section.
+// ShardPoolSize returns the  value of "shard_pool_size" config parameter from "storage" section.
```
Comment (carpawell) [inline cmd/neofs-node/config/engine/config.go:50]:
```suggestion
// ShardPoolSize returns the value of "shard_pool_size" config parameter from "storage" section.
```
here and below (think could be fixed with regex or in IDE)

File: pkg/innerring/innerring.go [hunk 11]
```diff
@@ -425,7 +425,7 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<-
 	}
 
 	if !server.sideNotaryConfig.disabled {
-		// enable notary support in the side client
+		// enable notary support in the sideclient
```
Comment (carpawell) [inline pkg/innerring/innerring.go:428]: i think one of the:
1. keep as it was done;
2. `sidechain client`.

File: pkg/innerring/processors/container/processor.go [hunk 12]
```diff
@@ -20,7 +20,7 @@ type (
 		IsAlphabet() bool
 	}
 
-	// Processor of events produced by container contract in morph chain.
+	// Processor of events produced by container contract in the morphchain.
```
Comment (carpawell) [inline pkg/innerring/processors/container/processor.go:23]: lets change that to `sidechain` to keep everything in sync

File: pkg/innerring/processors/neofs/process_assets.go [hunk 13]
```diff
@@ -8,12 +8,12 @@ import (
...
 const (
-	// lockAccountLifeTime defines amount of epochs when lock account is valid.
+	// lockAccountLifeTime defines the amount of epochs when lock account is valid.
 	lockAccountLifetime uint64 = 20
 )
 
-// Process deposit event by invoking balance contract and sending native
-// gas in morph chain.
+// Process deposit event by invoking a balance contract and sending native
+// gas in the morphchain.
```
Comment (carpawell) [inline pkg/innerring/processors/neofs/process_assets.go:16]:
```suggestion
// gas in the sidechain.
```

File: pkg/innerring/processors/governance/process_update.go [hunk 14]
```diff
@@ -123,7 +123,7 @@ func (gp *Processor) processAlphabetSync(txHash util.Uint256) {
 		}
 	}
 
-	// 4. Update NeoFS contract in main net.
+	// 4. Update NeoFS contract in the mainnet.
```
Comment (carpawell) [inline pkg/innerring/processors/governance/process_update.go:126]: i would also change all "mainnet" to "mainchain", but not sure

File: pkg/morph/client/netmap/config.go [hunk 15]
```diff
@@ -55,7 +55,7 @@ func (c *Client) AuditFee() (uint64, error) {
 	return fee, nil
 }
 
-// EpochDuration returns number of sidechain blocks per one NeoFS epoch.
+// EpochDuration returns number of side chain blocks per one NeoFS epoch.
```
Comment (carpawell) [inline pkg/morph/client/netmap/config.go:58]:
```suggestion
// EpochDuration returns number of sidechain blocks per one NeoFS epoch.
```

File: pkg/morph/client/subnet/client.go [hunk 16]
```diff
@@ -48,7 +48,7 @@ func (x *InitPrm) SetBaseClient(base *client.Client) {
 	x.base = base
 }
 
-// SetContractAddress sets address of Subnet contract in NeoFS sidechain.
+// SetContractAddress sets address of Subnet contract in NeoFS side chain.
```
Comment (carpawell) [inline pkg/morph/client/subnet/client.go:51]:
```suggestion
// SetContractAddress sets address of Subnet contract in NeoFS sidechain.
```

File: pkg/morph/client/subnet/client.go [hunk 17]
```diff
@@ -69,7 +69,7 @@ const (
 	NotaryNonAlphabet
 )
 
-// SetMode makes client to work with non-notary sidechain.
+// SetMode makes client to work with non-notary side chain.
```
Comment (carpawell) [inline pkg/morph/client/subnet/client.go:72]:
```suggestion
// SetMode makes client to work with non-notary sidechain.
```

**Review by carpawell — COMMENTED** [review]
also, conflicts appeared

**Review by fyrchik — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/morph/internal/types.go (hunk 9 above)
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/internal/types.go:11]: Simple `Returns ...` is quite common https://github.com/golang/go/blob/master/src/cmd/link/internal/loader/loader.go#L752

File: pkg/innerring/innerring.go (hunk 11 above)
Comment (EliChin) [reply pkg/innerring/innerring.go:428]: Oh, thanks, my typo

File: cmd/neofs-node/config/engine/config.go (hunk 10 above)
Comment (EliChin) [reply cmd/neofs-node/config/engine/config.go:50]: Checked everywhere

File: pkg/innerring/processors/governance/process_update.go (hunk 14 above)
Comment (EliChin) [reply pkg/innerring/processors/governance/process_update.go:126]: Let's ask someone else, I'm not the person to decide upon it.
@fyrchik @realloc @alexvanin

Comment (realloc) [reply pkg/innerring/processors/governance/process_update.go:126]: Let's keep mainnet here. Otherwise, we risk to confuse everyone with a new term.

**Review by carpawell — APPROVED** [review]

**Pull request merged**